test/config.tests.js

```diff
@@ -38,19 +38,33 @@ describe('config', function() {
     });
     const config = getConfig(customConfig);
 
-    it('should default to response_type=id_token', function() {
+    it('should set new response_type', function() {
       assert.equal(config.authorizationParams.response_type, 'code');
     });
 
-    it('should default to response_mode=form_post', function() {
+    it('should allow undefined response_mode', function() {
       assert.equal(config.authorizationParams.response_mode, undefined);
     });
 
-    it('should default to scope=openid profile email ', function() {
+    it('should keep default to scope', function() {
```

Comment:
I'm assuming some miswording here; if so, here's a suggestion for ease.

```suggestion
    it('should keep default scope', function() {
```